cartography/intel/aws/organizations.py

```diff
@@ -24,7 +25,8 @@ def get_current_aws_account_id(boto3_session):
 
 def get_aws_account_default(boto3_session):
     try:
-        return {"default": get_current_aws_account_id(boto3_session)}
+        profile_name = os.getenv("AWS_PROFILE", default="default") or "default"
+        return {profile_name: get_current_aws_account_id(boto3_session)}
```

Comment:
Could change to;

```python
return {boto3_session.profile_name: get_current_aws_account_id(boto3_session)}
```

An then instantiate the `boto3_session` in `__init__.py` with;

```python
profile_name = os.getenv("AWS_PROFILE", default="default") or "default"
boto3_session = boto3.Session(profile_name=profile_name)
```

Reply:
I like your second idea better: defining `profile_name` adjusting `boto3_session` in `__init__.py`.